src/test/resources/com/atlassian/performance/tools/report/jstat/jstat.log

```diff
@@ -871,3 +871,4 @@
 2018-01-18T14:27:39+00:00          1739.1   0.00  99.81  16.83  92.70  90.98  84.06    377   24.676     9    6.112   30.788
 2018-01-18T14:27:41+00:00          1741.1   0.00  99.81  17.07  92.70  90.98  84.06    377   24.676     9    6.112   30.788
 2018-01-18T14:27:43+00:00          1743.2   0.00  99.81  17.07  92.70  90.98  84.06    377   24.676     9    6.112   30.788
+2019-01-18T15:10:32+00:00             0.7   0.00   0.00  69.23   0.00      -      -      0    0.000     0    0.000   0.000
```

Comment:
Thank you for expanding the test coverage to include the new behavior :)
If we want to be even more next level, we add the test first, in a separate commit. Then we can see that the test was red at first and actually does reproduce the problem in JPERF-426. Then we'll see that the next commit makes it green, proving that the code is indeed a fix :)

Reply:
TDD ;)